tutorials/ros_ign_demo.md

```diff
@@ -0,0 +1,75 @@
+# ROS Integration
+
+In this tutorial we will learn how to Integrate ROS with Ignition. We will establish communication between them. This can help in many aspects, we can receive data or commands from ROS and apply it to ignition and vice versa.
```

Comment:
```suggestion
In this tutorial we will learn how to Integrate ROS with Ignition. We will establish communication between them. This can help in many aspects, we can receive data or commands from ROS and apply it to Ignition and vice versa.
```